### PR TITLE
Default for translation

### PR DESCRIPTION
**Pull request by michaelneale**
# Description
Not ure if this is appropriate, but https://issues.jenkins-ci.org/browse/JENKINS-41114 is getting really annoying.

@reviewbybees 

can I get an amen?

### CODE REVIEW

File: blueocean-personalization/src/main/js/components/DashboardCards.jsx [hunk 1]
```diff
@@ -94,7 +94,9 @@ export class DashboardCards extends Component {
           cards={pausedCards}
         />) : null;
         const favoriteCardsStack = (<StackOutput
-          message={t('dashboardCard.input.favorite')}
+          message={t('dashboardCard.input.favorite', {
```
Comment (cliffmeyers) [inline blueocean-personalization/src/main/js/components/DashboardCards.jsx]: I believe there is another card stack that may need a default value for its label as well.

Comment (michaelneale) [inline blueocean-personalization/src/main/js/components/DashboardCards.jsx]: I'll take a look

**Comment by michaelneale** [comment]
I think AMERICAN spelling favorite is required

File: blueocean-personalization/src/main/js/components/DashboardCards.jsx (hunk 1 above)
Comment (michaelneale) [inline blueocean-personalization/src/main/js/components/DashboardCards.jsx]: @cliffmeyers I didn't see any others with that type of key

Comment (cliffmeyers) [inline blueocean-personalization/src/main/js/components/DashboardCards.jsx]: Here on line 93

File: blueocean-personalization/src/main/js/components/DashboardCards.jsx [hunk 2]
```diff
@@ -94,7 +94,9 @@ export class DashboardCards extends Component {
           cards={pausedCards}
```
Comment (cliffmeyers) [inline blueocean-personalization/src/main/js/components/DashboardCards.jsx]: Sorry, here it is. Should probably have a default too.

Comment (michaelneale) [inline blueocean-personalization/src/main/js/components/DashboardCards.jsx]: Don't follow - that is the file I changed?

Comment (cliffmeyers) [inline blueocean-personalization/src/main/js/components/DashboardCards.jsx]: https://gyazo.com/0718d4136a623ec3925a394139ebb09d

See above. You have specified a default value for `favoritesCardStack` but not for `pausedCardStack`

**Comment by michaelneale** [comment]
who cares about triggering...

File: blueocean-personalization/src/main/js/components/DashboardCards.jsx [hunk 3]
```diff
@@ -86,15 +86,19 @@ export class DashboardCards extends Component {
                     {cards}
                 </TransitionGroup>
             </div>);
-        };
+        };        
         // Only show paused pipelines when we really have some
         // do we have any paused pipelines?
         const pausedCardsStack = pausedCards.length > 0 ? (<StackOutput
-          message={t('dashboardCard.input.required')}
+          message={t('dashboardCard.input.favorite', {
```
Comment (scherler) [inline blueocean-personalization/src/main/js/components/DashboardCards.jsx]: you have switch the key here :bug:

Comment (michaelneale) [inline blueocean-personalization/src/main/js/components/DashboardCards.jsx]: what is this magic "switch key" you speak of?

**Comment by scherler** [comment]
BTW https://github.com/jenkinsci/blueocean-plugin/pull/692/files#diff-ff033537e38783f7671ed54ac3dcb687 it is fixed there

File: blueocean-personalization/src/main/js/components/DashboardCards.jsx (hunk 3 above)
Comment (scherler) [inline blueocean-personalization/src/main/js/components/DashboardCards.jsx]:
```
-          message={t('dashboardCard.input.required')}
+          message={t('dashboardCard.input.favorite', {
```

dashboardCard.input.required !== dashboardCard.input.favorite

**Comment by michaelneale** [comment]
@scherler ACK - I will leave this PR open as it serves as a demo of a webhook problem. I won't merge it.

File: blueocean-personalization/src/main/js/components/DashboardCards.jsx (hunk 3 above)
Comment (scherler) [inline blueocean-personalization/src/main/js/components/DashboardCards.jsx]: should have said "switched" ;)

**Pull request closed without merging**